### PR TITLE
Add the "planning-scheduler" for Angular

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: packages/angular/angular.json [hunk 1]
```diff
@@ -86,8 +86,8 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "7mb",
-                  "maximumError": "8mb"
+                  "maximumWarning": "8mb",
+                  "maximumError": "9mb"
```
Comment (ivanblinov2k17) [inline packages/angular/angular.json:90]: why this needs to be raised?

File: packages/angular/src/app/services/data.service.ts [hunk 2]
```diff
@@ -131,4 +132,85 @@ export class DataService {
 
     return from([items])
   };
+
+  public getListDS = () => from([[
```
Comment (ivanblinov2k17) [inline packages/angular/src/app/services/data.service.ts:136]: why do you need to dublicate service? can't you just use dx-template-gallery package functions?

File: packages/angular/angular.json (hunk 1 above)
Comment (GoodDayForSurf) [reply packages/angular/angular.json:90]: I think it's cos we have started using scheduler module.

File: packages/angular/src/app/services/data.service.ts (hunk 2 above)
Comment (GoodDayForSurf) [reply packages/angular/src/app/services/data.service.ts:136]: add dependency for only one service? Besides, it seems we should to use "angular way" services based on rxjs.

**Review by ivanblinov2k17 — APPROVED** [review]

File: packages/angular/src/app/pages/planning-scheduler/planning-scheduler.component.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    <div class='scheduler-container'>
+      <dx-scheduler #schedulerRef
+                    [adaptivityEnabled]="!!((screen.screenChanged | async)?.isXSmall)"
+                    [allDayPanelMode]="'hidden'"
+                    [dataSource]="tasks"
+                    [currentView]="currentView"
+                    [currentDate]="getSchedulerCurrentDate | apply: currentDate"
+                    (currentViewChange)="onCurrentViewChange($event)"
+                    height='inherit'
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-scheduler/planning-scheduler.component.html:30]: looks unstructured. Could you please structure properties like: simple binding, one-way binding and then emits?

File: packages/angular/src/app/pages/planning-scheduler/planning-scheduler.component.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        </dxi-resource>
+        <dxi-view type='day'></dxi-view>
+        <dxi-view type='workWeek'></dxi-view>
+        <dxi-view type='month'></dxi-view>
+        <dxi-view type='agenda'></dxi-view>
+      </dx-scheduler>
+      <dx-tooltip #tooltipRef
+                  *ngIf="selectedAppointment"
+                  [target]="selectedAppointment?.target"
+                  showEvent='click'
```
Comment (dxvladislavvolkov) [inline packages/angular/src/app/pages/planning-scheduler/planning-scheduler.component.html:50]: The same - unstructured

**Pull request merged**